Sources/PostgresNIO/New/PSQLError.swift

```diff
@@ -490,7 +583,11 @@ extension PostgresDecodingError: CustomStringConvertible {
         // This may seem very odd... But we are afraid that users might accidentally send the
         // unfiltered errors out to end-users. This may leak security relevant information. For this
         // reason we overwrite the error description by default to this generic "Database error"
-        "Database error"
+        """
+        PostgresDecodingError – This error description is generic on purpose to prevent leakage of
+        impl. details to upstream services. For debugging details use String(reflecting: error) on
+        PostgresDecodingError.
+        """
```

Comment:
I don't love a multiline message here, that's liable to make quite a muddle of peoples' logs.
```suggestion
        """
        PostgresDecodingError – Generic description to prevent accidental leakage of sensitive data. For details, use `String(reflecting: error)`.
        """
```
I know, it's longer on the one line, but it's barely 1.5x as long as one line of yours.